curriculum/challenges/english/25-front-end-development/quiz-loops-and-sequences/67f41268a129e63f8e071736.md

```diff
@@ -17,439 +17,522 @@ To pass the quiz, you must correctly answer at least 18 of the 20 questions belo
-Placeholder distractor 3
+You will receive a `TypeError`.
 
 #### --answer--
 
-Placeholder answer
+You will receive an `IndexError`.
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+What will be printed to the console?
+
+```py
+desserts = ['Cake', 'Cookies', 'Ice Cream', 'Pie']
+print(desserts[1:3]) # ?
+```
 
 #### --distractors--
 
-Placeholder distractor 1
+`['Cake', 'Cookies', 'Ice Cream', 'Pie']`
 
 ---
 
-Placeholder distractor 2
+`['Cake', 'Cookies']`
 
 ---
 
-Placeholder distractor 3
+`['Cake', 'Cookies','Cookies', 'Ice Cream']`
 
 #### --answer--
 
-Placeholder answer
+`['Cookies', 'Ice Cream']`
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+Which of the following methods is used to add an item to the end of a list?
 
 #### --distractors--
 
-Placeholder distractor 1
+`add()`
 
 ---
 
-Placeholder distractor 2
+`pop()`
 
 ---
 
-Placeholder distractor 3
+`push()`
 
 #### --answer--
 
-Placeholder answer
+`append()`
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+What will be printed to the console?
+
+```py
+numbers = [1, 2, 3, 4, 5]
+even_numbers = [6, 8, 10]
+
+numbers.extend(even_numbers)
+print(numbers) # ?
+```
 
 #### --distractors--
 
-Placeholder distractor 1
+`[2, 3, 4, 5, 6, 8, 10]`
 
 ---
 
-Placeholder distractor 2
+`[5, 6, 8, 10]`
 
 ---
 
-Placeholder distractor 3
+`[1, 2, 3, 4]`
 
 #### --answer--
 
-Placeholder answer
+`[1, 2, 3, 4, 5, 6, 8, 10]`
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+Which of the following methods is used to insert an element at a specific index in a list?
 
 #### --distractors--
 
-Placeholder distractor 1
+`push()`
 
 ---
 
-Placeholder distractor 2
+`add()`
 
 ---
 
-Placeholder distractor 3
+`place()`
 
 #### --answer--
 
-Placeholder answer
+`insert()`
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+What will be printed to the console?
+
+```py
+numbers = [1, 2, 3, 4, 5, 5, 5]
+numbers.remove(5)
+
+print(numbers) # ?
+```
 
 #### --distractors--
 
-Placeholder distractor 1
+`[1, 2, 3, 4]`
 
 ---
 
-Placeholder distractor 2
+`[1, 2, 3, 4, 5, 5, 5]`
 
 ---
 
-Placeholder distractor 3
+`[1, 2, 3, 4, 5]`
 
 #### --answer--
 
-Placeholder answer
+`[1, 2, 3, 4, 5, 5]`
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+Which of the following methods is used to empty a list?
 
 #### --distractors--
 
-Placeholder distractor 1
+`pop()`
 
 ---
 
-Placeholder distractor 2
+`empty()`
 
 ---
 
-Placeholder distractor 3
+`remove()`
 
 #### --answer--
 
-Placeholder answer
+`clear()`
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+What is a tuple?
 
 #### --distractors--
 
-Placeholder distractor 1
+It is a mutable data type of a random sequence of values.
 
 ---
 
-Placeholder distractor 2
+It is a special data type used to represent large numbers.
 
 ---
 
-Placeholder distractor 3
+It is a method used to add elements to a list.
 
 #### --answer--
 
-Placeholder answer
+It is an ordered sequence of values.
```

Comment:
```suggestion
It is an immutable ordered sequence of values.
```
This makes clear where lays difference between lists and tuples. Otherwise the answer still could be used for similar question, but regarding lists.